applications/matter_bridge/Kconfig

```diff
@@ -98,6 +98,30 @@ config BT_GATT_CLIENT
 config BT_GATT_DM
 	default y
 
+config BT_SMP
+	default y
+
+if BT_SMP
+
+config BT_SETTINGS
+	default y
+
+# Configure how many Bluetooth LE devices will be paired with Matter bridge. Assuming it will be only bridged devices, it should be set to BT_MAX_CONN - 1.
```

Comment:
It was my plan as well, but I think unfortunately it can't. It is because we redefine the default value of `BT_MAX_CONN` above and then we would try to use it for calculation, but it is done before sourcing Zephyr root Kconfig. So at this stage the `BT_MAX_CONN` config has no type defined and cannot be used for calculation as non-integer value.